pipelines/templates/build-test-publish.yaml

```diff
@@ -394,8 +387,12 @@ stages:
           inputs:
             targetType: 'inline'
             script: |
-              make DOCKER_REGISTRY=$REGISTRY_NAME PRESIDIO_LABEL=$(Build.BuildID) test-functional-no-build
-            workingDirectory: '$(MODULEPATH)'
+              make DOCKER_REGISTRY=$REGISTRY_NAME PRESIDIO_LABEL=$(Build.BuildID) TEST_IN_CONTAINER=true test-functional-no-build
+        - task: PublishTestResults@2
```

Comment:
you can "name" this test results with _testRunTitle_ to separate them from other tests that might run (not sure have those in this project)

Reply:
done in the container